### PR TITLE
FindFolder and monitor custom folder events

### PR DESCRIPTION
**Pull request by tomjebo**
This is a simple change to add custom folder event monitoring.

### CODE REVIEW

**Comment by v-am** [comment]
MailSim/MailSim.v12.suo probably should not be committed. Maybe we should delete it from the repository.

I think it would be better to implement FindFolder as a more general method on MailStore class. It would either get one of default folders or try to find a subfolder with the specified name.

The name comparison while looking for the folder should be case-insensitive.

**Comment by tomjebo** [comment]
Yes, I agree that the suo needs to be removed. 
I'll take a look at MailStore for FindFolder and the case sensitivity.

File: MailSim/MailSim/ExecuteSequence.cs [hunk 1]
```diff
@@ -742,9 +742,17 @@ private bool EventMonitor(MailSimOperationsEventMonitor operation)
                 MailFolder folder = olMailStore.GetDefaultFolder(operation.Folder);
                 if (folder == null)
                 {
-                    Log.Out(Log.Severity.Error, operation.OperationName, "Unable to retrieve folder {0}",
-                     operation.Folder);
-                    return false;
+                    // might be a custom folder under the Inbox (right now we'll only support folders under the Inbox)
+                    MailFolder inboxFolder = olMailStore.GetDefaultFolder("olFolderInbox");
```
Comment (PuiL-Msft) [inline MailSim/MailSim/ExecuteSequence.cs]: this is hard coded to look only under inbox, we should generalize it, may be just search thru the supported folder? other ideas?

Comment (tomjebo) [inline MailSim/MailSim/ExecuteSequence.cs]: This is definitely a simple first stab.  I was thinking Inbox subfolders was the most likely  scenario but a more general approach might be better for more flexible testing scenarios, as Alex mentioned.

**Comment by AndrewDavidoff** [comment]
I also agree with Alex's suggestions.

**Pull request closed without merging**